### PR TITLE
docs(tutorial): Added import in Jokes tutorial

### PR DESCRIPTION
**Pull request by sidv1905**
Added import statement of useLoaderData in jokes tutorial docs.

### CODE REVIEW

**Comment by coindegen** [comment]
duplicate of https://github.com/remix-run/remix/pull/1568

**Comment by sidv1905** [comment]
Oops ! I did not see it in the issues, will take it down. Thanks !

**Comment by machour** [comment]
@sidv1905 your PR was here first, so it will get merged.
Could you sign the CLA by adding your name to the `contributors.yml` file ?

**Comment by sidv1905** [comment]
@machour , CLA signed , thanks for the approval

File: contributors.yml [hunk 1]
```diff
@@ -279,3 +279,4 @@
 - youngvform
 - zachdtaylor
 - zainfathoni
+- sidv1905
```
Comment (machour) [inline contributors.yml:282]: We're trying to keep this sorted alphabetically, could you move it up ? ⬆️

Comment (sidv1905) [reply contributors.yml]: Done

**Review by machour — APPROVED** [review]

File: docs/tutorials/jokes.md [hunk 2]
```diff
@@ -1582,8 +1582,10 @@ import type { LoaderFunction } from "remix";
 import type { User } from "@prisma/client";
 
 import { db } from "~/utils/db.server";
+import { useLoaderData } from "remix";
```
Comment (MichaelDeBoey) [inline docs/tutorials/jokes.md:1585]: We already have an import statement for `remix` here, so please add `useLoaderData` in that line (L2)

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

File: docs/tutorials/jokes.md (hunk 2 above)
Comment (machour) [reply docs/tutorials/jokes.md:1585]: It's an `import type`

Comment (MichaelDeBoey) [reply docs/tutorials/jokes.md:1585]: We have

```tsx
// this is just an example. No need to copy/paste this 😄
import { json } from "remix";
import type { LoaderFunction } from "remix";
```

So `useLoaderData` can be added to the `json` import line

Comment (machour) [reply docs/tutorials/jokes.md:1585]: oh, totally didn't see that

Comment (machour) [reply docs/tutorials/jokes.md:1585]: @sidv1905 do you mind fixing this last thing? I swear we're gonna get this merged 😅

**Pull request merged**